index.js

```diff
@@ -167,11 +167,22 @@ async function bundle(items, options) {
 		or a timestamped file (for the moment) 
 		in case we're bundling many web pages.
 	 */
-	const output_path =
-		options.output ||
-		(items.length === 1
-			? `${slugify(items[0].title || 'Untitled page')}.pdf`
-			: `percollate-${Date.now()}.pdf`);
+	let output_path;
+
+	if (options.individual && options.output) {
+		if (!fs.existsSync(options.output)) {
+			fs.mkdirSync(options.output, { recursive: true });
+		}
+		output_path = `${options.output}/${slugify(
```

Comment:
About this: I wasn't specific in the original issue, but what I had in mind was to do something like:

```js
`${options.output.replace(/\.pdf$/i, '')}-${slugify(...)}.pdf`
```

So that, when called with:

```bash
percollate --output some.pdf --individual page1 page2 page3
```

You'd get:

```bash
some-page1.pdf
some-page2.pdf
some-page3.pdf
```

(Notice I'm stripping the `.pdf` extension from `--output` and then adding it back in after the page name).

Reply:
Sounds good. Will work on it.